lib/config/configuration.go

```diff
@@ -202,6 +202,10 @@ type CommandLineFlags struct {
 	// IntegrationConfEICEIAMArguments contains the arguments of
 	// `teleport integration configure eice-iam` command
 	IntegrationConfEICEIAMArguments IntegrationConfEICEIAM
+
+	// IntegrationConfAWSOIDCIdPArguments contains the arguments of
+	// `teleport integration configure awsoidc-idp` command
+	IntegrationConfAWSOIDCIdPArguments IntegrationConfAWSOIDCIdP
```

Comment:
sorry I couldn't help but only complain here that the names are looong. The elephant in the room probably is the word `Integration` and something like `AWSOIDCIdPConfigArgs` would have been much better? 
The ship has sailed and you can ignore this comment....

Reply:
Yeah
Mixed feelings, honestly.
We could change all of them
```
IntegrationConfDeployServiceIAMArguments to AWSOIDCDeployServiceConfigArgs

IntegrationConfEICEIAMArguments to AWSOIDCEICEConfigArgs

IntegrationConfAWSOIDCIdPArguments to AWSOIDCIdPConfigArgs
```

Or none :sweat_smile: 

What do you think?


:hot_face: 
AWSOIDCEICEConfigArgs
Amazon Web Services OpenID Connect Ec2 Instance Connect Endpoint Configuration Arguments

Reply:
> AWSOIDCEICEConfigArgs, Amazon Web Services OpenID Connect Ec2 Instance Connect Endpoint Configuration Arguments

that's brutal 😅. Btw, if we put them behind `awsoidcidp` package, then we can simply name it `Ec2InstanceConnectConfigArgs`. The caller will have enough detail `awsoidcidp.Ec2InstanceConnectConfigArgs`.

Re: to update or not - Tbh, I am not sure if the change is worth the effort as this requires changes to code already merged. I just had to call this out!

Reply:
I think I'll leave it like this for now, then